packages/devtools_app/test/app_size_screen_test.dart

```diff
@@ -84,8 +84,8 @@ void main() {
       appSizeController = AppSizeTestController();
       fakeServiceManager = FakeServiceManager();
       setGlobal(ServiceConnectionManager, fakeServiceManager);
-      when(fakeServiceManager.errorBadgeManager.errorCountNotifier(any))
-          .thenReturn(ValueNotifier<int>(0));
+      // when(fakeServiceManager.errorBadgeManager.errorCountNotifier(any))
```

Comment:
why is this commented out

Reply:
Forgot to uncomment. Excluded this file from PR to handle it separately.